.ci/E2E-tests/execute.sh

```diff
@@ -45,15 +49,29 @@ export HOST_HOSTNAME=$(hostname)
 cd docker
 #just pull everything else than artemis-app as we build it later either way
 if [ "$TEST_FRAMEWORK" = "playwright" ]; then
-  echo "Building for playwright"
-  docker compose -f $COMPOSE_FILE pull artemis-playwright $CONFIGURATION nginx
-  docker compose -f $COMPOSE_FILE build --build-arg WAR_FILE_STAGE=external_builder --no-cache --pull artemis-app
-  docker compose -f $COMPOSE_FILE up --exit-code-from artemis-playwright
+  if [ "$CONFIGURATION" = "multi-node" ]; then
+    echo "Building for playwright (multi-node)"
+    docker compose -f $COMPOSE_FILE pull artemis-playwright $CONFIGURATION nginx
+    docker compose -f $COMPOSE_FILE build --build-arg WAR_FILE_STAGE=external_builder --no-cache --pull artemis-app-node-1 artemis-app-node-2 artemis-app-node-3
```

Comment:
I guess if you wanted, you could only keep the build command in the if else as the other two are the same, but its not too important (to me)

Same below